godel_msgs/msg/BlendingPlanParameters.msg

```diff
@@ -1,12 +1,16 @@
 # Path Parameters
-float64 tool_radius             # (m) actual radius of blending tool
+float64 tool_radius           # (m) actual diameter of blending tool
 float64 margin                  # (m) additional distance to leave near surface boundaries
 float64 overlap                 # (m) overlap distance between adjacent paths
 
+# Process Parameters
+float64 spindle_speed           # (rpm) Spindle Speed
+float64 tool_force					# (kg)  Downward force to apply
```

Comment:
Sorry about all these indentation errors. I will convert components to be style-guide-compliant as I run into mixed indentation files.